special-pages/pages/new-tab/app/next-steps/components/NextSteps.module.css

```diff
@@ -89,6 +107,27 @@
     }
 }
 
+.confirmation {
+    display: flex;
+    align-items: center;
+    transition: all .2s ease-in;
+    font-size: calc(11 * var(--px-in-rem));
+    line-height: calc(14 * var(--px-in-rem));
+    min-height: 26px;
+    
+    svg {
+        height: 1rem;
+        width: 1rem;
+        margin-right: var(--sp-2);
+    }
+    
+    p {
+        max-width: 8rem;
+        font-weight: 600;
+        font-size: calc(11 * var(--px-in-rem));
```

Comment:
This is already set above, and will inherit I believe

Reply:
good catch. moved the text-specific styling down to this p tag section